src/InfrastructureSystems.jl

```diff
@@ -75,7 +75,7 @@ Subtypes may contain time series. Which requires
 All subtypes must include an instance of ComponentUUIDs in order to track
 components attached to each attribute.
 """
-abstract type InfrastructureSystemsSupplementalAttribute <: InfrastructureSystemsType end
+abstract type SupplementalAttribute <: InfrastructureSystemsType end
```

Comment:
I renamed this struct because it needs to get imported and re-exported in PowerSystems. I think we want users to type this name instead.

Reply:
Ok, the old name was under the premise that the interface in PSY was different and attributed was not used directly